Source/Core/DolphinNoGUI/MainNoGUI.cpp

```diff
@@ -139,19 +139,18 @@ bool Host_RendererIsFullscreen()
 
 void Host_ConnectWiimote(int wm_idx, bool connect)
 {
-  if (Core::IsRunning() && SConfig::GetInstance().bWii &&
-      !SConfig::GetInstance().m_bt_passthrough_enabled)
-  {
-    Core::QueueHostJob([=] {
-      bool was_unpaused = Core::PauseAndLock(true);
-      const auto bt = std::static_pointer_cast<IOS::HLE::Device::BluetoothEmu>(
-          IOS::HLE::GetDeviceByName("/dev/usb/oh1/57e/305"));
-      if (bt)
-        bt->AccessWiiMote(wm_idx | 0x100)->Activate(connect);
-      Host_UpdateMainFrame();
-      Core::PauseAndLock(false, was_unpaused);
-    });
-  }
+  const auto ios = IOS::HLE::GetIOS();
+  if (!ios || SConfig::GetInstance().m_bt_passthrough_enabled)
+    return;
+  Core::QueueHostJob([=] {
+    bool was_unpaused = Core::PauseAndLock(true);
+    const auto bt = std::static_pointer_cast<IOS::HLE::Device::BluetoothEmu>(
+        ios->GetDeviceByName("/dev/usb/oh1/57e/305"));
```

Comment:
Nope, looks like a mistake. I've moved it to inside the lambda to avoid a race condition (since GetIOS() runs on the CPU thread, I think, and the pointer is used on the host thread sometime in the future).